### PR TITLE
finish 8 slides

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/js/templates/FourthBlock.js [hunk 1]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
+        let video = document.querySelector(`.Fourth-block.e-tron .video-container video[data-slide="${dataValue}"]`)
+        let playStatus = document.querySelector(`.Fourth-block.e-tron .progress__play-status[data-slide="${dataValue}"]`)
+        let playStatusPaused = document.querySelector(`.Fourth-block.e-tron .progress__play-status--pause[data-slide="${dataValue}"]`)
+        let playStatusPlayed = document.querySelector(`.Fourth-block.e-tron .progress__play-status--play[data-slide="${dataValue}"]`)
+
+        if (item.classList.contains('active')) {
+            video.play()
+            item.classList.remove('active')
+            playStatus.classList.add('animated')
+            playStatusPaused.beginElement()
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: move out from `if {} else {}`

File: src/js/templates/FourthBlock.js [hunk 2]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
 })
+
+// e-tron video
+fourthBlockInnerCircle && fourthBlockInnerCircle.forEach(item => {
+    item.classList.add('active')
+})
+
+fourthBlockInnerCircle && fourthBlockInnerCircle.forEach(item => {
+    item.addEventListener('click', () => {
+        let dataValue = item.dataset.slide
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: change all `let` to `conts`

File: src/js/templates/FourthBlock.js [hunk 3]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
+            playStatusPlayed.beginElement()
+        }
+    })
+})
+
+
+fourthBlockFullScreenButton && fourthBlockFullScreenButton.forEach(item => {
+    item.addEventListener('click', () => {
+        const delay = 300
+        let dataValue = item.dataset.slide
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: change all `let` to `conts`

File: src/js/templates/FourthBlock.js [hunk 4]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
+            }, delay)
+        }
+    })
+})
+
+
+fourthBlockProgressPlayStatus && fourthBlockProgressPlayStatus.forEach(item => {
+    item.addEventListener('click', () => {
+        const delay = 300
+        let dataValue = item.dataset.slide
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: change all `let` to `conts`

File: src/js/templates/FourthBlock.js [hunk 5]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
+
+
+//As video progresses, seekBar moves forward
+fourthBlockPlayer && fourthBlockPlayer.forEach(item => {
+    let dataValue = item.dataset.slide
+    let progress = document.querySelector(`.Fourth-block.e-tron .progress__seekbar div[data-slide="${dataValue}"]`)
+    let progressCircle = document.querySelector(`.Fourth-block.e-tron .progress__seekbar--circle[data-slide="${dataValue}"]`)
+
+    item.addEventListener('timeupdate', function updateProgress() {
+        let value = (100 / item.duration) * item.currentTime + '%'
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: use ES6 string

File: src/js/templates/FourthBlock.js [hunk 6]
```diff
@@ -48,3 +55,116 @@ closeThreeDimensionView && closeThreeDimensionView.addEventListener('click', ()=
...
+
+        progress.style.width = value
+        progressCircle.style.left = value
+
+        if (progress.style.width != '100%') {
```
Comment (StadniukYevhen) [inline src/js/templates/FourthBlock.js]: use strict `!==`

File: src/js/templates/FourthBlock.js (hunk 2 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: done

File: src/js/templates/FourthBlock.js (hunk 1 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: not needs

File: src/js/templates/FourthBlock.js (hunk 3 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: done

File: src/js/templates/FourthBlock.js (hunk 4 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: done

File: src/js/templates/FourthBlock.js (hunk 5 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: done

File: src/js/templates/FourthBlock.js (hunk 6 above)
Comment (nmelnychenko) [reply src/js/templates/FourthBlock.js]: done

**Pull request merged**